rustls/Cargo.toml

```diff
@@ -36,6 +36,11 @@ zlib-rs = { version = "0.2", optional = true }
 default = ["aws_lc_rs", "logging", "std", "tls12"]
 std = ["webpki/std", "pki-types/std", "once_cell/std"]
 logging = ["log"]
+# XXX TODO NAMING - ADD SEPARATORS
+# XXX TODO ADD THIS TO DEFAULT FEATURES
+defaultproviderenabled = []
+# XXX TODO NAMING - ADD SEPARATORS
+withrcalias = []
```

Comment:
Can we actually do this with features? It causes a public API breaking change. Perhaps we can use `#[cfg(target_has_atomic = "ptr")]`, which simply echoes the non-portability of Arc?

Reply:
I would totally like to get rid of the new features, do have a couple questions:

Any pointers or ideas how we might test this in CI (and on a local workstation)?

I would also love to add an option to keep the Arc support working with the portable-atomics crate. This could increase the complexity of the cfg conditions in multiple places. We could try to address this at some point in the future, with a little extra code churn.